Cargo.toml

```diff
@@ -11,5 +11,6 @@ serde = { version = "1.0", features = ["derive"] }
 itertools = { version = "0.14" }
 thiserror = { version = "2.0.17" }
 tracing = { version = "0.1" }
+derive_more = { version = "2", features = ["from"] }
```

Comment:
Are you actually using this anywhere? thiserror also has a 'from' macro. I don't know if we need derive_more